src/core/index.js

```diff
@@ -7,21 +7,23 @@ const PeerBook = require('peer-book')
 
 const defaultRepo = require('./default-repo')
 
-const goOnline = require('./ipfs/go-online')
-const goOffline = require('./ipfs/go-offline')
-const isOnline = require('./ipfs/is-online')
-const load = require('./ipfs/load')
-const version = require('./ipfs/version')
-const id = require('./ipfs/id')
-const repo = require('./ipfs/repo')
-const init = require('./ipfs/init')
-const bootstrap = require('./ipfs/bootstrap')
-const config = require('./ipfs/config')
-const block = require('./ipfs/block')
-const object = require('./ipfs/object')
-const libp2p = require('./ipfs/libp2p')
-const files = require('./ipfs/files')
-const bitswap = require('./ipfs/bitswap')
+const goOnline = require('./modules/go-online')
```

Comment:
Why the renaming to `modules`. I agree `ipfs` isn't a great name, but `modules` isn't describing it much better. Maybe `commands`?